src/components/MarketTable/index.tsx

```diff
@@ -30,6 +30,7 @@ const TableHeadCell = tw.div`
 `;
 
 const MarketTable: React.FC<MarketTableProps> = ({ coins, loading, error }) => {
+  console.log(coins, loading, error);
```

Comment:
no

Reply:
Gone. This was from creating a branch off PR 37 instead of master.